tests/message-extensions/src/card.ts

```diff
@@ -161,6 +161,16 @@ export async function createDummyCards(searchQuery: string) {
       thumbnail: {
         title: item.title,
         text: item.description,
+        // When a user clicks on a list item in Teams:
+        // - If the thumbnail has a `tap` property: Teams will trigger the `message.ext.select-item` activity
+        // - If no `tap` property: Teams will insert the full adaptive card into the compose box
+        // tap: { 
+        //   type: "invoke",
+        //   title: item.title,
+        //   value: {
+        //     "option": index,
+        //   },
+        // },
```

Comment:
Should this be split into a separate snippet? Or extend cards with a `const cardsWithSelect`?

Reply:
Even though it's a bit messy, I think having the snippet close to where it belongs is easier for learning purposes.